src/components/FilterDivider.tsx

```diff
@@ -0,0 +1,12 @@
+import { twMerge } from "../hooks/useComposedCssClasses";
+/**
+ * A divider component for NumericalFacets, HierarchicalFacets, and StaticFacets.
```

Comment:
this is used in `StandardFacets` (not `StaticFacets`) and should be placed between `StaticFilters` components